events-to-histogram/src/main.rs

```diff
@@ -48,6 +55,9 @@ struct Cli {
 
     #[clap(long)]
     time_end: Time,
+
+    #[clap(long, env, default_value = "127.0.0.1:9091")]
+    metrics_address: SocketAddr,
```

Comment:
Why add this? There is already `observability_address` above that was used previously.